### PR TITLE
Fix the issue that HTTP headers except for Content-Type are ignored.

### PR DESCRIPTION
**Pull request by yuokada**

### CODE REVIEW

File: airframe-http-finagle/src/test/scala/wvlet/airframe/http/finagle/FinagleRouterTest.scala [hunk 1]
```diff
@@ -65,6 +65,14 @@ trait MyApi extends LogSupport {
     Future.value(request.toString)
   }
 
+  @Endpoint(path = "/v1/json_header_test")
+  def jsonHeaderTest(): HttpMessage.Response = {
```
Comment (xerial) [inline airframe-http-finagle/src/test/scala/wvlet/airframe/http/finagle/FinagleRouterTest.scala:69]: httpHeaderTets?

Comment (yuokada) [reply airframe-http-finagle/src/test/scala/wvlet/airframe/http/finagle/FinagleRouterTest.scala:69]: Ah... will fix.

**Comment by yuokada** [comment]
@takezoe or @shimamoto Could you take a look if you get a chance?

**Comment by yuokada** [comment]
Yep. Thanks for your help!

**Pull request merged**